src/plugins/drawing/objects/drawing-object.ts

```diff
@@ -65,6 +66,13 @@ export const DrawingObject = types.model("DrawingObject", {
     // self then MobX observation is not triggered so moving the element doesn't
     // cause the selection highlight to update.
     throw "Subclass needs to implement this";
+  },
+  get description(): string {
```

Comment:
I would probably call this `displayName` or `label` or something similar. I think of a sentence or two when I think of "description".